<launch>
    <node name="atxmegaTalker" pkg="robotBrain" type="atxmegaTalker" />
</launch>